stdlib/http/src/main/ballerina/http/http_secure_client.bal

```diff
@@ -277,23 +278,23 @@ public function createHttpSecureClient(string url, ClientEndpointConfig config)
 #
 # + req - An HTTP outbound request message
 # + config - Client endpoint configurations
-# + return - Prepared HTTP request or `error` if an error occurred at auth handler invocation
+# + return - Prepared HTTP request or `http:ClientError` if an error occurred at auth handler invocation
 function prepareSecureRequest(Request req, ClientEndpointConfig config) returns Request|error {
```

Comment:
Any reason why have we returned `error` instead of return type of prepareAuthenticationError method?

Reply:
It's a mistake. Will update.